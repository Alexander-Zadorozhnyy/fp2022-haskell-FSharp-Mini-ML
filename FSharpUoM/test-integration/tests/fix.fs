let rec fix f x = f (fix f) x; 
fix (fun re n -> if n <= 1 then 1 else n * re (n - 1)) 5;;
