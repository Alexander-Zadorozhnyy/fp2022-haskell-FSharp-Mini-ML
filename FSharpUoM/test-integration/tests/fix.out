val fix: forall a2 a3. (a2 -> a3) -> a3
val it: int -> int
val it: int = 3628800
